spec/requests/request_spec.rb

```diff
@@ -347,6 +347,16 @@
     end
   end
 
+  describe 'location metadata' do
```

Comment:
I don't think I'd expect to find this test in this file. How did you decide to put it here?

Reply:
This is where the test for the identifiers was - seemed like where the catalog show specs are.